src/sr/2018-01/07/05.md

```diff
@@ -1,7 +1,21 @@
 ---
-title:  Lekcija
+title:  Poštene dessetak: svet Gospodu
```

Comment:
Since this section of the file is `yaml` the title in this case has to be wrapped with `'` since there is semicolon. Ex: 
`title: '...'`